src/elisp/treemacs-core-utils.el

```diff
@@ -923,11 +917,11 @@ successful.
 PATH: Filepath | Node Path
 PROJECT Project Struct"
   (treemacs-with-path path
-    :file-action (when (file-exists-p path)
-                   (treemacs-find-file-node path project))
-    :top-level-extension-action (treemacs--find-custom-top-level-node path)
-    :directory-extension-action (treemacs--find-custom-dir-node path)
-    :project-extension-action (treemacs--find-custom-project-node path)))
+                      :file-action (when (file-exists-p path)
```

Comment:
Huh?

Reply:
Oops, Emacs hadn't picked up the `(declare (indent 1))` when I started editing